src/lua/popen.c

```diff
@@ -2529,6 +2554,12 @@ tarantool_lua_popen_init(struct lua_State *L)
 		lua_pushstring(L, popen_lua_actions[i].value);
 		lua_setfield(L, -2, popen_lua_actions[i].name);
 	}
+	lua_newtable(L);
+	{
```

Comment:
Why this `{` block is needed?

Reply:
That block is a result of the inspiration of other similar modules with Lua C API in Tarantool.